Sidebar_Suggestions.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
```

Comment:
May be include this CSS for google styling -> https://ssl.gstatic.com/docs/script/css/add-ons.css